code/modules/organs/organ_internal.dm

```diff
@@ -109,6 +109,8 @@
 				take_damage(1, prob(30))
 
 /datum/internal_organ/proc/take_damage(amount, silent= FALSE)
+	if(SSticker.mode?.flags_round_type & MODE_NO_PERMANENT_WOUNDS)
```

Comment:
You'll probably end up finding a couple of sources which give organ damage directly instead of through take_damage but those can be fixed as they show up and should be changed anyway.

Reply:
Looks like everything uses take_damage appart from flashes, which is fine by me